SunriseTests/ViewModels/ProductViewModelSpec.swift

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright (c) 2016 Commercetools. All rights reserved.
+//
+
+import Quick
+import Nimble
+import ObjectMapper
+import ReactiveCocoa
+import Result
+@testable import Sunrise
+
+class ProductViewModelSpec: QuickSpec {
+
+    override func spec() {
+        describe("ProductViewModel") {
+            var productViewModel: ProductViewModel!
+
+            beforeEach {
+                let path = NSBundle.currentTestBundle!.pathForResource("product-projection", ofType: "json")!
+                let productProjectionJSON = try! NSString(contentsOfFile: path, encoding: NSUTF8StringEncoding)
+                let product = Mapper<ProductProjection>().map(productProjectionJSON)!
+
+                productViewModel = ProductViewModel(product: product)
+            }
+
+            it("has the correct upper case name") {
+                expect(productViewModel.name).to(equal("SNEAKERS ”TOKYO” LOTTO GREY"))
```

Comment:
This tests fails since `NSLocale.currentLocale()` is set to German on my machine and name returns `"SNEAKERS ”TOKYO” LOTTO GRAU"`. Is there a way to set the currentLocale explicitly on the test-setup to be independent from the system environment?